app/components/status_tag/view.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
 class StatusTag::View < ViewComponent::Base
-  def initialize(trainee:)
+  def initialize(trainee:, css_classes: "")
```

Comment:
We have a standard pattern of supplying extra classes to view components - see `FormComponents::Autocomplete::View ` as an example. You should be able to add your `"trainee_status"` to `default_classes` and call `super` in the `initialize`.

Reply:
Thanks for your help with this!